source/common/http/codec_helper.h

```diff
@@ -7,6 +7,18 @@ namespace Http {
 
 class StreamCallbackHelper {
 public:
+  void runLowWatermarkCallbacks() {
+    for (StreamCallbacks* callbacks : callbacks_) {
+      callbacks->onBelowWriteBufferLowWatermark();
```

Comment:
Yeah that's what I meant. I think within the callback should work technically. If it doesn't I wouldn't fix in this change but I might drop a TODO in. (I'm kind of guessing it doesn't both in the stream and connection case, but not sure).